.github/workflows/make_wheel_macOS_arm64.sh

```diff
@@ -0,0 +1,22 @@
+set -e -x
+
+export TF_NEED_CUDA=0
+
+python --version
+python -m pip install --default-timeout=1000 delocate wheel setuptools tensorflow==$TF_VERSION
+
+python configure.py
+
+bazel build \
+  --cpu=darwin_arm64 \
+  --copt -mmacosx-version-min=11.0 \
+  --linkopt -mmacosx-version-min=11.0 \
```

Comment:
We don't need to support 10.13 since the only the latest version macOS version currently supports ARM64. But setting a minimum version here makes sure that the script will keep working even if the GitHub actions VM are updated to 11.1 at some point in the future.